File: Medium/1446 - angle-between-hands-of-a-clock/Solution.java
class Solution {
    public double angleClock(int hour, int minutes) {
        double diff = (hour * 1.0) + ((minutes * 1.0) / 60);
        diff -= (minutes * 1.0) / 5;

        diff = diff > 0 ? diff : -diff;
        diff *= 30;

        if(diff > 180) return 360 - diff;

        return diff;
    }
}

File: Medium/1446 - angle-between-hands-of-a-clock/Solution.cpp
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double hr = (hour%12) + minutes / 60.0;
        double mins = minutes/5.0;
        double angle = abs(hr-mins)*30;
        return min(angle, 360.0-angle);
    }
};